#32 Adds Script for Google Analytics and Ads to the <head> of All Pages

BigSteveLittle opened the pull request:
This pull request is in regard to issue #32 

Google Analytics is SaaS that records visitor behaviour while Google Ads is SaaS that places ads within the SERP (Search Engine Ranking Page). This issue adds the required script to all pages.

BigSteveLittle committed:
Adds Issue #32 google-tag to README.md

BigSteveLittle committed:
Adds Google Tag Script to All HTML <head>s

EngelbertHumperdinck reviewed (changes requested):
Hey Steve, seems there has been some (non-critical) white space added to the <body> HTML. You might want to tidy this up before uploading. Thanks, @EngelbertHumperdinck

BigSteveLittle committed:
Adds Google Tag ID: G-6E1GLCGHXH to <head> Element of All Pages

BigSteveLittle commented:
Hi Engelbert, I have cleaned up the HTML formatting of all srcset images and added the Google Tag code. Can you please do a final review and approve? Thanks, @BigSteveLittle

EngelbertHumperdinck reviewed (approved):
Hi @BigSteveLittle all looks good now @EngelbertHumperdinck

BigSteveLittle committed:
Updates the README.md File to Complete the Coding For the Google Tag
